tracks/ruby/exercises/clock/README.md

```diff
@@ -17,7 +17,7 @@ class Clock
   end
 
   def to_s
-    "%02d:%02d" % time.divmod(MINUTES_PER_HOUR)
+    format("%02d:%02d", *time.divmod(MINUTES_PER_HOUR))
```

Comment:
The style guide is only a guide.  I prefer the original line as less noisy.  Yes, somewhat mystical the first time you see it.  Since we are talking about "Style Guide" I am thinking Rubocop here.  It would also suggest not using double quotes, I believe.  The single quotes help re-inforce that this is not some trick of "string interpolation" but rather substitution.

Reply:
I looked at the code when I dropped in, not realizing that I had already responded to the preference.  :blush: :rosette: :

Reply:
Changing to single quotes is an awesome idea! I _did_ think it was a string interpolation trick, the first time I saw it. 👍 The single quotes are really meaningful here.